packages/babel-preset-env/src/index.ts

```diff
@@ -170,6 +170,11 @@ export const getModulesPluginNames = ({
     modulesPluginNames.push("syntax-top-level-await");
```

Comment:
The other syntax plugins can also be removed in Babel 8, since we can't disable them anyway, regardless of whether the caller actually supports certain features or not.